acceptance_test.go

```diff
@@ -0,0 +1,183 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package vitess implements Vitess connector for Conduit.
+// It provides both, a source and a destination Vitess connector.
+
+package vitess
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/brianvoe/gofakeit"
+	"github.com/conduitio-labs/conduit-connector-vitess/config"
+	"github.com/conduitio-labs/conduit-connector-vitess/source"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/matryer/is"
+	"go.uber.org/goleak"
+	"vitess.io/vitess/go/vt/vitessdriver"
+)
+
+var (
+	queryCreateTestTable  = `create table %s (id int, name text, primary key(id));`
+	queryCreateTestVindex = `alter vschema on %s add vindex hash(id) using hash;`
+	queryDropTestTable    = `drop table %s;`
```

Comment:
IIUC, if a test failed early on (e.g. connection issue), there will be no table at all. In that case, a clean-up will fail, right?

Reply:
I've added `if exists` to the query.